assets/js/googlesitekit/datastore/user/key-metrics.test.js

```diff
@@ -527,9 +544,68 @@ describe( 'core/user key metrics', () => {
 					coreKeyMetricsEndpointRegExp
 				);
 			} );
+
+			it( 'should return the filtered widget slugs that do not require custom dimensions when the user is in a view-only dashboard', () => {
+				// Set up state to simulate view-only mode
```

Comment:
Please finish comments with a full stop :)

```suggestion
				// Set up state to simulate view-only mode.
```